### ISSUE TITLE
Can this be built to work in React?

### ISSUE THREAD

**Issue by asha23** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#287]
I only use React.

**Comment by slbug** [comment]
No, definitely no! This hipster's react has nothing similar to our beloved java!

**Comment by emiln** [comment]
It might be worth looking into running React in JavaFX and prerendering with Nashorn. Then React could use Redux for state management and talk to FBEE over SOAP. This sounds like a solid idea and we'll happily accept pull requests implementing it.
